database/migrations/postgres/1720012750850077_rename_commit_events.up.sql

```diff
@@ -1,2 +1,6 @@
-ALTER TABLE IF EXISTS events
-    RENAME TO commit_events;
+DO $$
+BEGIN
+    IF NOT EXISTS (SELECT 1 FROM information_schema.tables WHERE table_name = 'commit_events') THEN
+        ALTER TABLE events RENAME TO commit_events;
```

Comment:
I think we should still keep the `IF EXISTS` part after the ALTER TABLE

Reply:
Done